compute/admin_guide/install/system_requirements.adoc

```diff
@@ -421,27 +483,41 @@ Prisma Cloud supports the official releases of the following orchestrators for t
 |===
 |Orchestrator |Version
 
-|OpenShift 4
-|OpenShift 4.10 (CRI-O 1.23)
-
-|EKS
-|EKS v1.21.5 (containerd 1.4.6)
-
-|ECS
+|Elastic Container Service (ECS)
 |AMI name: amzn2-ami-ecs-hvm-2.0.20220411-arm64-ebs
 
 ECS agent 1.61.0
 
 Docker 20.10.7
 
-|Kubernetes
+|Elastic Kubernetes Service (EKS)
+|EKS v1.21.5 (containerd 1.4.6)
+
+|GKE on ARM
+|GKE version: v1.24.1-gke.1400
+
+containerd version: 1.6.2
+
+[NOTE]
+====
+Defenders running in GKE on ARM don't support the following features:
+
+* Prevent for processes
+* File system events
```

Comment:
@rcaballeromx, we support both runtime detections for processes and file system, though the prevent capability doesn't work for both of them. From what you have written here it seems like we don't support file system at all.

Reply:
@MorShabiPAN I've changed the text to reflect that functionality and explicitly call out that we support runtime detection.